### PR TITLE
Add missing test parts(PendingResponse checks)

### PR DESCRIPTION
**Pull request by Jakub89**

### CODE REVIEW

File: concent_api/core/tests/test_integration_force_payment.py [hunk 1]
```diff
@@ -397,6 +402,9 @@ def test_provider_send_correct_force_payment_concent_should_accept(self):
         )
         self._assert_stored_message_counter_not_increased()
 
+        last_pending_message = PendingResponse.objects.filter(delivered=False).last()
```
Comment (rwrzesien) [inline concent_api/core/tests/test_integration_force_payment.py]: Please fix styling.

File: concent_api/core/tests/test_integration_force_payment.py [hunk 2]
```diff
@@ -378,6 +379,10 @@ def test_provider_send_correct_force_payment_concent_should_accept(self):
         )
         self._assert_stored_message_counter_not_increased()
 
+        last_pending_message = PendingResponse.objects.filter(delivered=False).order_by('created_at').last()
```
Comment (rwrzesien) [inline concent_api/core/tests/test_integration_force_payment.py]: Please fix styling.

File: concent_api/core/tests/test_auth_force_payment.py [hunk 3]
```diff
@@ -131,3 +136,6 @@ def test_provider_send_force_payment_and_concent_should_return_it_to_requestor_w
         self._assert_stored_message_counter_not_increased()
 
         self._assert_client_count_is_equal(1)
+
+        last_pending_message = PendingResponse.objects.filter(delivered=False).last()
```
Comment (rwrzesien) [inline concent_api/core/tests/test_auth_force_payment.py]: Please fix styling.

File: concent_api/core/tests/test_auth_force_payment.py [hunk 4]
```diff
@@ -88,6 +89,10 @@ def test_provider_send_force_payment_and_concent_should_return_it_to_requestor_w
         )
         self._assert_stored_message_counter_not_increased()
 
+        last_pending_message = PendingResponse.objects.filter(delivered=False).order_by('created_at').last()
```
Comment (rwrzesien) [inline concent_api/core/tests/test_auth_force_payment.py]: Please fix styling.

**Comment by Jakub89** [comment]
Ok, done

File: concent_api/core/tests/test_auth_report_computed_task.py [hunk 5]
```diff
@@ -953,8 +953,8 @@ def test_provider_forces_computed_task_report_and_requestor_does_not_respond_sho
 
         self.assertEqual(response.status_code,                                                                  200)
         self.assertEqual(force_report_computed_task_from_view.timestamp,                                        self._parse_iso_date_to_timestamp("2017-12-01 11:00:05"))
-        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute.timestamp,   self.force_report_computed_task.report_computed_task.task_to_compute.timestamp)
-        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute,             self.force_report_computed_task.report_computed_task.task_to_compute)
+        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute.timestamp,   self.force_report_computed_task.report_computed_task.task_to_compute.timestamp)  # pylint: disable=no-member
+        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute,             self.force_report_computed_task.report_computed_task.task_to_compute)  # pylint: disable=no-member
```
Comment (kbeker) [inline concent_api/core/tests/test_auth_report_computed_task.py]: If pylint warning are one by one make them to be in one straight line

Comment (Jakub89) [reply concent_api/core/tests/test_auth_report_computed_task.py]: Done.

File: concent_api/core/tests/test_auth_force_payment.py [hunk 6]
```diff
@@ -131,3 +136,6 @@ def test_provider_send_force_payment_and_concent_should_return_it_to_requestor_w
         self._assert_stored_message_counter_not_increased()
 
         self._assert_client_count_is_equal(1)
+
+        last_pending_message = PendingResponse.objects.filter(delivered = False).last()
+        assert last_pending_message is None
```
Comment (dybi) [inline concent_api/core/tests/test_auth_force_payment.py]: In tests, when you check results, I would use `self.assert*` from `unittest` or `assert_that` form `assertpy`

Comment (Jakub89) [reply concent_api/core/tests/test_auth_force_payment.py]: Done.

**Pull request merged**